pyfstat/__init__.py

```diff
@@ -40,3 +41,12 @@
 
 __version__ = get_versions()["version"]
 del get_versions
+
+try:
+    logger = set_up_logger()
+    logger.info(f"Running PyFstat version {__version__}")
```

Comment:
So this differs from #441 in that we always spin up our own in addition, even if the user preconfigured something else already...?

Reply:
The `_get_default_logger` logic sounds good, I'll test it a bit but just from reading it sounds like a good replacement for my stuff.